samples/openapi3/client/petstore/dart2/petstore_client_lib_fake/lib/model/enum_arrays.dart

```diff
@@ -159,7 +159,7 @@ class EnumArraysArrayEnumEnum {
   const EnumArraysArrayEnumEnum._(this.value);
 
   /// The underlying value of this enum member.
-  final List<String> value;
```

Comment:
@agilob This commit also fixes the wrong value type that was appearrent in the other PR.